Fix binary ops to return BOOLEAN

justinbowes opened the pull request:
Use newer "BOOL" ComfyUI convention for binary ops in other data types.

Fixes #9

justinbowes committed:
Rename "BOOL" to "BOOLEAN" everywhere

justinbowes reviewed (commented):
Change is working in my fork.

HerlanderCarvalho commented:
> Change is working in my fork.

You might want to change the README.md, because it is still pointing to the original one, and I only noticed it today, so I was cloning the wrong repository, and continued to have the same problem.

justinbowes commented:
> > Change is working in my fork.
> 
> You might want to change the README.md, because it is still pointing to the original one, and I only noticed it today, so I was cloning the wrong repository, and continued to have the same problem.

I do want to do that. Thank you.

justinbowes committed:
Update README with new repo link and fork note

Updated README to reflect the new repository link and added a note about the fork.

justinbowes commented:
> > Change is working in my fork.
> 
> You might want to change the README.md, because it is still pointing to the original one, and I only noticed it today, so I was cloning the wrong repository, and continued to have the same problem.

Sorry for the confusion. The intent in my repo was fork-to-merge, but since it seems long-lived, I've updated the README to credit `evanspearman` but also provide corrected installation instructions.

justinbowes commented:
Since my fork now updates README.md to provide correct installation instructions, and that wouldn't be correct for this repo, I'm closing this MR. Feel free to copy the relevant changes yourself.
